lib/components/ModalPortal.js

```diff
@@ -18,194 +18,218 @@ var CLASS_NAMES = {
   }
 };
 
-var ModalPortal = module.exports = React.createClass({
-
-  displayName: 'ModalPortal',
-  shouldClose: null,
+export default class ModalPortal extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    onAfterOpen: PropTypes.func,
+    closeTimeoutMS: PropTypes.number,
+    shouldCloseOnOverlayClick: PropTypes.bool,
+    onRequestClose: PropTypes.func,
+    className: PropTypes.string,
+    overlayClassName: PropTypes.string,
+    defaultStyles: PropTypes.shape({
+      content: PropTypes.object,
+      overlay: PropTypes.object
+    }),
+    style: PropTypes.shape({
+      content: PropTypes.object,
+      overlay: PropTypes.object
+    }),
+    role: PropTypes.string,
+    children: PropTypes.node,
+    contentLabel: PropTypes.string
+  };
+
+  static defaultProps = {
+    style: {
+      overlay: {},
+      content: {}
+    }
+  };
 
-  getDefaultProps: function() {
-    return {
-      style: {
-        overlay: {},
-        content: {}
-      }
-    };
-  },
+  static afterClose = () => {
```

Comment:
Should be preferable to declare as function instead of assign a anonymous function to a variable?

```javascript
static afterClose() {...}
```